belt/src/services/compilers/solc.ts

```diff
@@ -43,9 +43,7 @@ function compiler(
 ) {
   const _d = d.extend('compiler')
   // remove our custom versions property
-  const compilerSettingCopy: config.App['compilerSettings'] = JSON.parse(
-    JSON.stringify(compilerSettings),
-  )
+  const compilerSettingCopy: any = JSON.parse(JSON.stringify(compilerSettings))
```

Comment:
Would be nice if we could keep the typing here.

Reply:
Aha, did this because if we type it then the next line:

```javascript
delete compilerSettingCopy.versions
```

Will give off this warning/error: "The operand of a 'delete' operator must be optional."